lib/internal/source_map/prepare_stack_trace.js

```diff
@@ -64,14 +64,14 @@ const prepareStackTrace = (globalThis, error, trace) => {
         const {
           originalLine,
           originalColumn,
-          originalSource
+          originalSource,
+          name
```

Comment:
nit: consider adding a trailing comma to make future PR diffs cleaner?
```suggestion
          name,
```

Reply:
I think this is a style that a separate PR should use linting to enforce, and also apply everywhere. (iow i think it’s a great change, but i don’t think node’s followed it consistently thus far)

Reply:
Agreed, to be clear I'm +1 landing this with or without the trailing commas.